src/pytorch_lightning/accelerators/hpu.py

```diff
@@ -39,9 +39,24 @@ def setup_environment(self, root_device: torch.device) -> None:
             raise MisconfigurationException(f"Device should be HPU, got {root_device} instead.")
 
     def get_device_stats(self, device: Union[str, torch.device]) -> Dict[str, Any]:
-        """HPU device stats aren't supported yet."""
-        rank_zero_debug("HPU device stats aren't supported yet.")
-        return {}
+        """Returns a map of the following metrics with their values:
+
+        Limit,
```

Comment:
Can this refer to the HPU docs instead of listing them? So it doesn't become outdated

Reply:
Document link is provided in the doc. For the current version, this is what is supported. We dont have a design in place to upgrade the package separately.